pandas_datareader/yahoo/daily.py

```diff
@@ -97,7 +98,9 @@ def yurl(symbol):
 
     def _get_params(self, symbol):
         unix_start = int(time.mktime(self.start.timetuple()))
-        unix_end = int(time.mktime(self.end.timetuple()))
+        day_end = self.end + datetime.timedelta(days=1) \
```

Comment:
@jreback I'm proposing this change, because I noted that existing Yahoo tests behaved differently locally vs on Travis. @bashtage also seem to have observed same. I see existing tests covering this change (although maybe somebody could reconfirm boundaries?)